projects/aca-content/src/lib/components/search/search-input.service.ts

```diff
@@ -23,16 +23,33 @@
  * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { NgModule } from '@angular/core';
-import { CommonModule } from '@angular/common';
-import { CoreModule } from '@alfresco/adf-core';
-import { AppHeaderComponent } from './header.component';
-import { AppSearchInputModule } from '../search/search-input.module';
-import { AppToolbarModule } from '../toolbar/toolbar.module';
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
 
-@NgModule({
-  imports: [CommonModule, CoreModule.forChild(), AppSearchInputModule, AppToolbarModule],
-  declarations: [AppHeaderComponent],
-  exports: [AppHeaderComponent]
+@Injectable({
+  providedIn: 'root'
 })
-export class AppHeaderModule {}
+export class SearchInputService {
+  savedRoute = '';
```

Comment:
should be made private if no component is using it

Reply:
It is being used by SearchInputComponent